packages/teleport/src/services/auth/auth.ts

```diff
@@ -30,13 +30,14 @@ import { DeviceType } from './types';
 
 const auth = {
   u2fBrowserSupported() {
-    if (window['u2f']) {
-      return null;
+    if (!window['u2f'] || navigator.userAgent.includes('Chrome')) {
+      return new Error(
+        'The U2F API for hardware keys is not supported by your browser. \
+        Please notify your system administrator to update cluster settings \
+        to use WebAuthn as the second factor protocol. In the meantime, \
+        you can use Firefox to log in.'
```

Comment:
Looks good to me!

Reply:
nit: `In the meantime, you can use Firefox` or `In the meantime, you can use Firefox to use this hardware key`, since it's possible for this message to show up when the user is logged in (if they're first registering a key).

Reply:
good catch, the language also does not make sense when "creating" account as well. How about this:

`In the meantime, Firefox can be used as an alternative.`

Reply:
got a second opinion from zac, will go with your suggestion